core/new-gui/src/app/workspace/component/left-panel/operator-menu/operator-menu.component.spec.ts

```diff
@@ -106,14 +103,6 @@ describe("OperatorPanelComponent", () => {
     expect(component.groupNamesOrdered.length).toEqual(mockOperatorGroup.length);
   });
 
-  it("should have all group names shown in the UI side panel", () => {
-    const groupNamesInUI = fixture.debugElement
-      .queryAll(By.css(".texera-workspace-operator-panel-subgroup"))
-      .map(el => el.nativeElement.querySelector(".ant-collapse-header").innerText);
-
-    expect(groupNamesInUI).toEqual(mockOperatorGroup.map(group => group.groupName));
-  });
-
```

Comment:
can we keep the test case but modify it?

Reply:
Fixed.